src/Altinn.Platform/Altinn.Platform.Storage/UnitTest/TestingControllers/InstancesControllerGETTests.cs

```diff
@@ -0,0 +1,340 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+using Altinn.Authorization.ABAC.Xacml.JsonProfile;
+using Altinn.Common.PEP.Interfaces;
+using Altinn.Platform.Storage.Clients;
+using Altinn.Platform.Storage.Configuration;
+using Altinn.Platform.Storage.Controllers;
+using Altinn.Platform.Storage.Helpers;
+using Altinn.Platform.Storage.Interface.Models;
+using Altinn.Platform.Storage.Repository;
+using AltinnCore.Authentication.Constants;
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Controllers;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.AspNetCore.Routing;
+
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+using NSubstitute;
```

Comment:
All our test projects already have references to Moq and I would prefer if that was what we continued to use.

Reply:
Done!